models/public/alexnet/model.yml

```diff
@@ -32,7 +32,7 @@ files:
   - name: alexnet.caffemodel
     size: 243862414
     checksum: f102235bc9d8f422a04d5254a21fd1862986eae91c7337714fd5d78dcf37f835d05ed7943e80056ae19ca9e9c054c181
-    source: http://dl.caffe.berkeleyvision.org/bvlc_alexnet.caffemodel
```

Comment:
It makes sense to keep original URL somewhere, e.g. in `description` field.

Reply:
actually, I agree. @Wovchena @jkamelin can we replace source keyword with new, original_source keyword. So, when we upload public models package to storage.openvinotoolkit.org, CI will add us keywords source for actual link where we will download model by default, and original_source will be kept in model.yml files for reference. And potentially, we even may add option to downloader to get models from original links ?

Reply:
The original URL is kept in `source.txt` similar to how it was done for 2021.4 public models

Reply:
yeah, but having this as a keyword in model.yml allows to use it for downloading as an option. and this way we keep all info in single place

Reply:
Done